src/stores/AGBStore.ts

```diff
@@ -0,0 +1,9 @@
+import { defineStore } from "pinia";
+
+export const AGBStore = defineStore('agb',{
```

Comment:
Maybe we could rename the store into "Payment" or so to use it not only for the AGBs.